libraries/plugins/json_rpc/json_rpc_plugin.cpp

```diff
@@ -246,11 +255,21 @@ void json_rpc_plugin_impl::process_params( string method, const fc::variant_obje
          vector< std::string > v;
          boost::split( v, method, boost::is_any_of( "." ) );
 
-         FC_ASSERT( v.size() == 2, "method specification invalid. Should be api.method" );
+         FC_ASSERT( v.size() == 2 || v.size() == 3, "method specification invalid. Should be api.method or network.api.method" );
+
+         if( v.size() == 2 ) {
+            api_name = v[ 0 ];
+            method_name = v[ 1 ];
+            func_args = request.contains("params") ? request[ "params" ] : fc::json::from_string("{}");
+            if( func_args.is_object() && func_args.get_object().contains("network_id") )
+               network_name = func_args.get_object()["network_id"].as<std::string>();
```

Comment:
do we want to support two different ways of specifying the same parameter ? Might be ambiguos for users which one is applied if they specify different network in "method" and param "network_id"

Reply:
there are already multiple methods how to call an rpc method:
```
{"jsonrpc":"2.0", "method":"call", "params":["database_api", "get_dynamic_global_properties"], "id":1}
{"jsonrpc":"2.0", "method":"call", "params":["database_api", "get_dynamic_global_properties", {}], "id":2}
{"jsonrpc":"2.0", "method":"database_api.get_dynamic_global_properties","id":4}
{"jsonrpc":"2.0", "method":"database_api.get_dynamic_global_properties", "params":{}, "id":5}
```
a bit of flexibility probably won't hurt...

Reply:
In last commit, also options 
```
{"jsonrpc":"2.0", "method":"call", "params":["network_id", "database_api", "get_dynamic_global_properties"], "id":1} 
{"jsonrpc":"2.0", "method":"call", "params":["network_id", "database_api", "get_dynamic_global_properties", {} ], "id":2}
```
were added